packs/slack/actions/list_channels.yaml

```diff
@@ -0,0 +1,13 @@
+---
+  name: "list_channels"
+  runner_type: "run-python"
+  description: "List channels in a Slack Org"
+  enabled: true
+  entry_point: "list_channels.py"
+  parameters:
+    token:
+      type: "string"
+      description: "Slack API token."
+    exclude_archived:
```

Comment:
This should be a boolean and we should send a correct integer inside the action.